modules/understanding-the-compliance-scan-status.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * manage-compliance/managing-compliance-20.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="understanding-the-compliance-scan-status_{context}"]
+= Understanding the compliance scan status
+
+By understanding the compliance scan status, you can manage the overall security posture of your environment.
+
+[cols="2,2",options="header"]
+|===
+|Status |Description
+
+|`Fail`
+|The compliance check failed.
+
+|`Pass`
+|The compliance check passed.
+
+|`Not Applicable`
+|{product-title-short} skipped the check because it was not applicable.
```

Comment:
Do we want to consider leaving RHACS out of this statement to be consistent with other entries in the table?

Also, the compliance operator is the engine evaluating applicability of the rules. RHACS is attempting to transparently present those results to users. Some readers might think RHACS is in control of the applicability of the check, when that's not necessarily the case.

Reply:
@dashrews78 ^^

Reply:
Yeah I agree with @rhmdnd on that one.  That status is just a pass through from the compliance operator.  RHACS is just the presenter of the information.